### PR TITLE
Add documentation for TriggerLatentClientEvent and TriggerLatentServerEvent

### PR DESCRIPTION
**Pull request by AvarianKnight**

### CODE REVIEW

File: content/docs/scripting-manual/working-with-events/triggering-events.md [hunk 1]
```diff
@@ -68,3 +89,33 @@ TriggerClientEvent(player, "eventName", eventParam1, eventParam2);
...
+Latent events take an extra paramater 'bps' which stands for 'bytes per second', this defines how fast it should send data to the server.
+
+**Lua**
+```lua
+TriggerLatentClientEvent("eventName", targetPlayer, bps, eventParam1, eventParam2)
+-- Use -1 for "targetPlayer" if you want the event to trigger on all connected clients.
+```
+
+**C#**
+```csharp
```
Comment (4mmonium) [inline content/docs/scripting-manual/working-with-events/triggering-events.md:106]: Is there a reason, for the `bps` arguments to not be passed in any of these examples? I was thinking that maybe there was an overload function somewhere, but I couldn't find any.

https://github.com/citizenfx/fivem/blob/f78dc3121f83c8f27eadee71e5d623ba835904d7/code/client/clrcore/BaseScript.cs#L264

**Review by 4mmonium — APPROVED** [review]
Thanks for resolving the missing `bps` argument for the C# example, great work :-)

**Comment by blattersturm** [comment]
@4mmonium request review/merge again? Someone'd been pointing this out :D

**Comment by 4mmonium** [comment]
> @4mmonium request review/merge again? Someone'd been pointing this out :D

I reviewed this all over again, just in case I missed something the last time I reviewed and I guess this is 'ok to merge' as is 😃

**Pull request merged**